src/htcondor_es/convert_to_json.py

```diff
@@ -569,7 +570,9 @@ def get_creation_time_from_taskname(ad):
     """
     try:
         _str_date = ad['CRAB_Workflow'].split(':')[0]
-        return int(time.mktime(datetime.datetime.strptime(_str_date, '%y%m%d_%H%M%S').timetuple()))
+        _naive_date = datetime.datetime.strptime(_str_date, '%y%m%d_%H%M%S')
```

Comment:
if we depend on the timestamp in the crab taskname in any way, and in particular on it being in a specific time zone, this needs to be very well made clear, documented, and propagated in the CRAB code, since atm AFAIK we have it there only to guarantee unique names for each user, so format and time zone could change.

Reply:
Actually, it would be better if it came as a classAd (instead of using the name in this way).  
It was discussed in https://github.com/dmwm/cms-htcondor-es/issues/10